src/main/java/com/sprint/mission/discodeit/controller/AuthController.java

```diff
@@ -35,10 +43,55 @@ public class AuthController {
   @PostMapping("/login")
   public ResponseEntity<UserDTO> login(
       @RequestBody @Valid AuthLoginRequest authLoginRequest) {
+
     UserDTO authLoginResponse = authService.login(authLoginRequest);
 
     return ResponseEntity
         .status(HttpStatus.OK)
         .body(authLoginResponse);
   }
+
+  @Operation(summary = "CSRF 토큰 발급 API", responses = {
+      @ApiResponse(responseCode = "203", description = "토큰 발급 성공"),
+      @ApiResponse(responseCode = "500", description = "서버 오류")
+  })
+  @GetMapping("/csrf-token")
+  public ResponseEntity<Void> getCsrfToken(CsrfToken csrfToken) {
+
+    String token = csrfToken.getToken();
+
+    log.debug("CSRF Token 요청 : {}", token);
+
+    return ResponseEntity.status(HttpStatus.NON_AUTHORITATIVE_INFORMATION)
```

Comment:
```suggestion
     return ResponseEntity
        .status(HttpStatus.NO_CONTENT)
        .build();
}
```

이게 더 의도에 맞는 코드일수도..!

Reply:
그렇네요 ㅎㅎ... 바꿔놓겟습니다!